src/components/quote/quote.tsx

```diff
@@ -12,6 +12,7 @@ const QuoteText = styled.blockquote`
   color: ${theme.palette.text.topline};
   padding-left: 20px;
   margin: 0;
+  box-shadow: inset 2px 0 0 ${theme.palette.text.topline};
```

Comment:
I wonder if this needs to be a `box-shadow`. Why not a simple `border-left`?
`box-shadow` has been known for it's bad performance on mobile devices... maybe that is outdated now but using a `box-shadow` to create a solid left border feels like it's overkill.

Reply:
Figma does it like that so I guess false friend 😁 I suggest to use a border left too.